server/bin/pbench-index.py

```diff
@@ -428,7 +473,7 @@ def main(options, name):
                         f"Logic Bomb!  Unexpected tar ball handling "
                         f"result status {tb_res:d} for tar ball {tb}"
                     )
-                elif tb_res >= 4 or res <= 11:
+                elif tb_res >= 4 or res < error_codes["GENERIC_ERROR"].value:
```

Comment:
This is a weird expression. I don't really see how it even makes sense; but this is the purpose for which I'd envisioned the "ignorable" property on the error object, in response to Pete's earlier comment.

If we can't figure out how to leverage that here, maybe the attribute was a bad idea.

Reply:
I agree with Dave, maybe instead of assigning a value to `res` or `tb_res` we can keep them as ErrorCode object so we can check if they are ignorable or not something like `if res.ignorable or tb_res.ignorable`. Please check my previous comment

Reply:
"ignorable" property just checks whether the code is ignorable or not while here we are dealing with error codes. We need to specify a range or error code for which a particular message assigned to it is logged. If I use ignorable here it might be true for multiple values and we might end up logging the wrong error message. like it might be true for GENERIC values as well but we do not deal with GENERIC values here.